WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -6,8 +6,8 @@ def index
       format.html
       format.js do
         persons = Person.in_region(params[:region]).order(:name)
-        params[:search]&.split&.each do |part|
-          persons = persons.where("MATCH(rails_persons.name) AGAINST (:name_match IN BOOLEAN MODE) OR wca_id LIKE :wca_id_part", name_match: "#{part}*", wca_id_part: "#{part}%")
+        params[:search]&.split&.select { |part| part.length >= 4 }.each do |part|
+          persons = persons.where("MATCH(rails_persons.name) AGAINST (:name_match IN BOOLEAN MODE) OR wca_id LIKE :wca_id_part OR ", name_match: "#{part}*", wca_id_part: "#{part}%")
```

Comment:
Did you mean to add the " OR " to the end of this line?

Reply:
No, I was actually wondering why that was there. I guess I added it by accident.